modules/47_regipol/regiCarbonPrice/datainput.gms

```diff
@@ -18,9 +18,17 @@ p47_implicitQttyTargetTaxRescale_iter("1", "2030",ext_regi,qttyTarget,qttyTarget
 $endIf.cm_implicitQttyTarget
 
 *** RR this should be replaced as soon as non-energy is treated endogenously in the model
-*** EUR in 2030 ~ 90Mtoe (90 * 10^6 toe -> 90 * 10^6 toe * 41.868 GJ/toe -> 3768.12 * 10^6 GJ * 10^-9 EJ/GJ -> 3.76812 EJ * 1 TWa/31.536 EJ -> 0.1194863 TWa) EU27 = 92% EU28"
-p47_nonEnergyUse("2030",ext_regi)$(sameas(ext_regi, "EUR_regi")) = 0.1194863;
-p47_nonEnergyUse("2030",ext_regi)$(sameas(ext_regi, "EU27_regi")) = 0.11;
+*** https://ec.europa.eu/eurostat/databrowser/bookmark/f7c8aa0e-3cf6-45d6-b85c-f2e76e90b4aa?lang=en
+*** EU27 -> (2018:2021)/4 = 3.835 EJ = 0.121606 Twa (2020 non-energy use)
+*** EU27 -> 2030 non energy use is 91.6% of 2020 in pm_fe_nechem.cs4r (92.4% for EU28)
+p47_nonEnergyUse("2030",ext_regi)$(sameas(ext_regi, "EU27_regi")) = 0.121606*0.916; 
+p47_nonEnergyUse("2030",ext_regi)$(sameas(ext_regi, "EUR_regi")) = 0.13*0.924;
+p47_nonEnergyUse("2050",ext_regi)$(sameas(ext_regi, "EU27_regi")) = 0.121606*0.815; 
+p47_nonEnergyUse("2050",ext_regi)$(sameas(ext_regi, "EUR_regi")) = 0.13*0.841;
+*** DEU -> 0.946 EJ = 0.03 Twa (2020 non-energy use)
+p47_nonEnergyUse("2030",ext_regi)$(sameas(ext_regi, "DEU")) = 0.03*0.928; 
```

Comment:
where do the 0.928 and 0.848 and 0.822 come from?

I guess the 2030 value is from the historical 2020 value (as in EU), but how is this projected to 2045/2050 - a REMIND run? 
(I am not doubting the values, I am just thinking about the next person who has to work on this in 2 years and is wondering "was there a good source for this, or can I simply change it?")

Reply:
`pm_fe_nechem` values were calculated ad hoc by Michaja, and refers originally to a very rough estimation of non-energy use in chemicals. 
Due the lack of a better alternative until we have proper non-energy trajectories being considered in REMIND, these are scaled to represent total non-energy use as described in: https://github.com/pik-piam/remind2/pull/411 .

Reply:
the values that you see multiplied by the 2020 original non-energy use, are simply the relative difference that the exogenously defined trajectory in `pm_fe_nechem` considers in comparison to 2020.